pkg/client/client.go

```diff
@@ -632,17 +631,17 @@ func (c *Client) CreateOrUpdateDeployment(dep *appsv1.Deployment) error {
 	if err != nil {
 		uErr, ok := err.(*apierrors.StatusError)
 		if ok && uErr.ErrStatus.Code == 422 && uErr.ErrStatus.Reason == metav1.StatusReasonInvalid {
-			// try to delete deployment
+			// try to delete Deployment
 			err = c.DeleteDeployment(dep)
 			if err != nil {
-				return errors.Wrap(err, "deleting deployment object failed")
+				return errors.Wrap(err, "deleting Deployment object failed")
 			}
 			err = c.CreateDeployment(dep)
 			if err != nil {
-				return errors.Wrap(err, "creating deployment object failed after update failed")
+				return errors.Wrap(err, "creating Deployment object failed after update failed")
 			}
 		}
-		return errors.Wrap(err, "updating deployment object failed")
+		return errors.Wrap(err, "updating Deployment object failed")
```

Comment:
Thanks :)

Reply:
Follow up task to us, maybe tech-debt, unrelated to this PR: we should include the name of these objects in the logs. WDYT?

Reply:
good idea, could you create JIRA?

Reply:
Done! https://issues.redhat.com/browse/MON-1048